samples/dfu/README.rst

```diff
@@ -124,7 +150,7 @@ Now build Zephyr sample application for nRF52840 DK:
 
 .. code-block:: console
 
-   $ west build -b nrf52840dk_nrf52840 samples/dfu
+   $ west build -b nrf52840dk_nrf52840 samples/dfu -p
```

Comment:
Why adding `-p` everywhere? I don't think that Zephyr docs include that parameter, so why should we be different? If you will convince Zephyr maintainers to include that parameter in every command in docs, then I won't object to include that here. Let's be consistent.